### ISSUE TITLE
Not working with onItemTouchListener

### ISSUE THREAD

**Issue by anoop-coolest** [traex/RippleEffect#40]
I have added a custom onItemTouchListener for reacting to touch events,that works well but ripple effect is all gone.Is there some sort of conflict between these two??I am pretty new to android so please dont mind if i am missing something really easy

**Comment by traex** [comment]
You're right, it's maybe a conflict between this two. Today I don't have a good solution, but I'm working on it ;)
